netci.groovy

```diff
@@ -9,7 +9,7 @@ def project = GithubProject
 def branch = GithubBranchName
 def isPR = true
 
-def platformList = ['Debian8.2:x64:Debug', 'Ubuntu:x64:Release', 'OSX:x64:Release', 'Windows_NT:x64:Release', 'Windows_NT:x86:Debug', 'RHEL7.2:x64:Release', 'CentOS7.1:x64:Debug']
+def platformList = ['Debian8.2:x64:Debug', 'Ubuntu:x64:Release', 'Ubuntu16.04:x64:Release', 'OSX:x64:Release', 'Windows_NT:x64:Release', 'Windows_NT:x86:Debug', 'RHEL7.2:x64:Release', 'CentOS7.1:x64:Debug', 'Fedora23:x64:Debug']
```

Comment:
We sprinkle Debug/Release into our existing platforms.

I think it would make sense to make one of the Ubuntu builds release and the other debug.  (similar to the way we do Windows)

---
In reply to: [64984469](https://github.com/dotnet/cli/pull/3278#discussion_r64984469) [](ancestors = 64984469)